src/components/Page.tsx

```diff
@@ -61,8 +65,10 @@ const Page: FunctionComponent<PageProps & HTMLAttributes<HTMLDivElement>> = ({
             data-role='page'
             className={`page ${className}`}
             data-title={title}
-            data-backbutton={`${isBackButtonEnabled}`}
-            data-menubutton={`${isMenuButtonEnabled}`}
+            data-backbutton={isBackButtonEnabled}
+            data-menubutton={isMenuButtonEnabled}
+            data-dom-cache={isDomCacheEnabled}
```

Comment:
Is this actually used for anything? I did a bit of searching but I didn't see anywhere that was reading the `data-dom-cache` attribute. :man_shrugging:

Reply:
yah your are right. I couldn't find it to so I removed it https://github.com/jellyfin/jellyfin-web/pull/3567/commits/328afb0db8dc7892ddd7522d98d6c89aa1379fb0